week-1/Homework/extra/2-piping.js

```diff
@@ -15,44 +14,53 @@
   3. Write a more readable version of what you wrote in step 2 under the BETTER PRACTICE comment. Assign
   the final result to the variable goodCode
 */
-
-function add() {
-
+let a, b, c, d;
+function add(a, b) {
+  let summa = Math.round((a + b) * 10) / 10;
+  return summa;
 }
```

Comment:
Thanks for giving the feedback! I tried to do the function add (a,b) {let summa=a+b; return summa;} But the result couldn't pass automatic test in 2-piping.js because it add 2.4 to 5.3 and summa equals 7,6(9). I agree that it looks strange, but it works for me.